docs/index.md

```diff
@@ -1,7 +1,7 @@
-# Lockbox Extension
+# Lockbox Extension {: .center }
 
 Click below to install the Lockbox extension:
 
-[install][install-link]{: .button-link}
+[install][install-link]{: .button-link .center }
```

Comment:
Right now, this is superfluous, since `.button-link` applies `text-align: center;` and `.center` isn't defined anyway.